doc/releases/changelog-0.35.0.md

```diff
@@ -185,23 +313,26 @@
   now strictly faster for JAX.
   [(#4963)](https://github.com/PennyLaneAI/pennylane/pull/4963)
 
+* PennyLane can now use lightning-provided VJPs by selecting `device_vjp=True` on the QNode.
+  [(#4914)](https://github.com/PennyLaneAI/pennylane/pull/4914)
+
 * `device_vjp` can now be used with normal Tensorflow. Support has not yet been added
   for `tf.Function` and Tensorflow Autograph.
   [(#4676)](https://github.com/PennyLaneAI/pennylane/pull/4676)
 
-* PennyLane can now use lightning provided VJPs by selecting `device_vjp=True` on the QNode.
-  [(#4914)](https://github.com/PennyLaneAI/pennylane/pull/4914)
-
-* Remove queuing (`AnnotatedQueue`) from `qml.cut_circuit` and `qml.cut_circuit_mc` to improve performance
+* Queueing has been removed (`AnnotatedQueue`) from `qml.cut_circuit` and `qml.cut_circuit_mc` to improve performance
   for large workflows.
   [(#5108)](https://github.com/PennyLaneAI/pennylane/pull/5108)
 
-* Improve the performance of circuit-cutting workloads with large numbers of generated tapes.
+* The performance of circuit-cutting workloads with large numbers of generated tapes have been improved.
   [(#5005)](https://github.com/PennyLaneAI/pennylane/pull/5005)
 
+* Measuring `qml.probs`is now faster due to an optimization in `_samples_to_counts`.
```

Comment:
```suggestion
* Measuring `qml.probs` is now faster due to an optimization in `_samples_to_counts`.
```

This one could also do with more details!

Reply:
@trbromley maybe we can remove the reference to the hidden function `_samples_to_counts` and explain the enhancement itself?